MomsSpaghetti/Pages/Index.cshtml

```diff
@@ -225,15 +225,31 @@
 
                                 @foreach (var ingredient in Model.Recipe.Ingredients)
                                 {
+                                    @*<li class="recipe__item">
+                    <svg class="recipe__icon">
+                        <use href="img/icons.svg#icon-check"></use>
+                    </svg>
+                    <div class="recipe__count">2</div>
+                    <div class="recipe__ingredient">
+                        <span class="recipe__unit">g</span>
```

Comment:
What's g?

Reply:
It is part of the code that I commented out from the original code. 2,g and pizza are hardcoded values inside the loop where the ingredients had to be displayed. I replaced this part with ingredient.ToString() so as to display the ingredients for the specific recipe.

Reply:
I forgot to remove the commented out part